api-processor/src/main/java/com/synopsys/integration/alert/processor/api/distribute/ProviderMessageDistributor.java

```diff
@@ -61,7 +61,7 @@ public void distribute(ProcessedNotificationDetails processedNotificationDetails
 
     public void distributeIndividually(UUID jobExecutionId, UUID jobId, String jobName, ChannelKey destinationKey, ProcessedProviderMessageHolder processedMessageHolder) {
         Set<Long> notificationIds = processedMessageHolder.extractAllNotificationIds();
-        auditAccessor.createOrUpdatePendingAuditEntryForJob(jobId, notificationIds);
+        //auditAccessor.createOrUpdatePendingAuditEntryForJob(jobId, notificationIds);
```

Comment:
tagging this comment in case we should remove it before merging

Reply:
will remove it in a cleanup refactor